app/assets/stylesheets/application.css

```diff
@@ -35,4 +35,10 @@
    font-size: 25px;
    color: #002244;
    font-family: "Marker Felt", fantasy;
- }
\ No newline at end of file
+ }
+
+ .scroll_bar{height:1400px;
+             overflow:auto;}
+
+
+
```

Comment:
Added some css to make a scroll bar. This needs to be a horizontal scroll bar in the future.

Reply:
Good work on the scrollbar! I'm excited to figure out how this front-end world works haha